lib/seahorn/BvOpSem2.cc

```diff
@@ -965,9 +998,14 @@ class OpSemMemManager {
   void onFunctionEntry(const Function &fn) {
     Expr res = m_ctx.read(m_sp0);
 
+    // align of semantic_word_size, or 4 if it's less than 4
+    unsigned offsetBits = 2;
+    switch (wordSzInBytes()) {
+      case 8: offsetBits = 3;
+    }
+    m_ctx.addDef(bv::bvnum(0, offsetBits, m_efac), bv::extract(offsetBits - 1, 0, res));
```

Comment:
this should probably be `addSide` and not `addDef`. I need to check on the difference in their semantics. Intuitively, `addDef` adds a definition, but the constraint added is not a definition...